trained_weights/LICENSE.md

```diff
@@ -0,0 +1,422 @@
+# MIT License
```

Comment:
We don't need the train weight license as we are not including it on the reopo. We have already mentioned in the README.md that the trained weight has MIT license

Reply:
> Thanks for the PR. Do we still need to keep the two images (grace hopper and kangaroo) or can we pull the two images instead? In this case, we can get rid of the submodule repo

I included both images as binary files here as they are only ~500kb each, so yes everything from the submodule repo is here now. I also removed the submodule as part of this PR.

> We don't need the train weight license as we are not including it on the reopo. We have already mentioned in the README.md that the trained weight has MIT license

Alright, I'll remove it!